org.eclipse.swt.gtk.linux.x86/src/org/eclipse/swt/internal/c/gtk.d

```diff
@@ -6275,6 +6281,17 @@ guint n_accels;
 _GtkAccelGroupEntry * priv_accels;
 }
 version(DYNLINK){
+    extern (C) @nogc nothrow {
+        alias TGTKgtk_icon_theme_get_default = GtkIconTheme* function();
+        alias TGTKgtk_icon_theme_lookup_by_gicon = GtkIconInfo* function(GtkIconTheme*, GIcon*, gint size, GtkIconLookupFlags);
+        alias TGTKgtk_icon_info_load_icon = GdkPixbuf* function(GtkIconInfo*, GError**);
+    }
+
+    __gshared extern(D) {
+        TGTKgtk_icon_theme_get_default gtk_icon_theme_get_default;
+        TGTKgtk_icon_theme_lookup_by_gicon gtk_icon_theme_lookup_by_gicon;
+        TGTKgtk_icon_info_load_icon gtk_icon_info_load_icon;
+    }
```

Comment:
Compared to the `mixin` just below this, I find this is easier to work with.

Reply:
The existing string mixin is to be compatible with D1. No need for that now.